PersonApi.Tests/V1/Factories/ResponseFactoryTest.cs

```diff
@@ -67,5 +70,78 @@ public void CanMapADomainPersonToAResponsePerson(bool hasDob)
             response.PersonTypes.Should().BeEquivalentTo(person.PersonTypes);
             response.Tenures.Should().BeEquivalentTo(person.Tenures?.Select(x => ResponseFactory.ToResponse(x)));
         }
+
+
+        // Test that tenures are ordered
+        // Active first - newest to oldest
+        // Inactive - newest to oldest
+
+        // create list of active tenures - random dates
+        // create list of inactive tenures - random dates
+
+        [Fact]
+        public void PersonResponseObjectToResponseWhenCalledReturnsTenuresInCorrectOrder()
```

Comment:
Very good test, but I would break it in two tests. One for actives, the other for inactives. It is a bit on the larger side :)

Reply:
Thanks. I've actually split the test into three.